routing.md

```diff
@@ -597,10 +597,13 @@ Of course, implicit binding is also possible when using controller methods. Agai
 
 ```php
 use App\Http\Controllers\UserController;
-use App\Models\User;
 
 // Route definition...
 Route::get('/users/{user}', [UserController::class, 'show']);
+```
+
+```php
+use App\Models\User;
 
 // Controller method definition...
 public function show(User $user)
```

Comment:
@aryanjoshidev Just fyi: As Laravel uses [Torchlight](https://torchlight.dev) for fences code block rendering, you can use the `filename` info string attribute
```suggestion
```php filename=routes/web.php
use App\Http\Controllers\UserController;

// Route definition...
Route::get('/users/{user}', [UserController::class, 'show']);
```

```php filename=app/Controllers/UserController.php
use App\Models\User;

// Controller method definition...
public function show(User $user)
```

Reply:
Technically, one could also resort to the use of tabs

````suggestion
```php tab="Route definition"
use App\Http\Controllers\UserController;

Route::get('/users/{user}', [UserController::class, 'show']);
```

```php tab="Controller method definition"
use App\Models\User;

public function show(User $user)
````